packages/visualizations-react/stories/Table/Table.stories.tsx

```diff
@@ -28,6 +28,12 @@ Playground.args = {
     options,
 };
 
+export const VerticalScroll = Template.bind({});
```

Comment:
So if we go for the 100% on the Table, we'll need to add a container with a fixed height (and maybe one with an `aspect-ratio`?) to match our use cases.

Reply:
The container was added to the story but I did not create a new story with`aspect-ratio` (I did not know the property)